tools/autograd/derivatives.yaml

```diff
@@ -1462,6 +1466,12 @@
 - name: mkldnn_convolution_backward(Tensor self, Tensor grad_output, Tensor weight, IntArrayRef padding, IntArrayRef stride, IntArrayRef dilation, int64_t groups, std::array<bool,3> output_mask)
   grad_output, self, weight: _convolution_double_backward(grads[0], grads[1], grads[2], grad_output, weight, self, stride, padding, dilation, false, std::vector<int64_t>(padding.size(), 0), groups, false, false, false, grad_input_mask)
 
+- name: mkldnn_linear(Tensor input, Tensor weight, Tensor bias)
+  input, weight, bias: mkldnn_linear_backward(input, grad, weight, grad_input_mask)
+
+- name: mkldnn_reshape(Tensor self, IntArrayRef shape)
```

Comment:
I'm surprised it does what you want, the method name in native_functions.yaml starts with underscore: _mkldnn_reshape

Reply:
solve it.